protocols/imap/src/main/java/org/apache/james/imap/api/message/MessageFlags.java

```diff
@@ -26,7 +26,7 @@
 import javax.mail.Flags;
 
 /**
- * The set of flags associated with a message. TODO - why not use
+ * The set of flags associated with a message.
  * javax.mail.Flags instead of having our own.
```

Comment:
It's the end of the sentence : "TODO - why not use javax.mail.Flags instead of having our own."